src/course-updates/delete-modal/DeleteModal.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import DeleteModal from './DeleteModal';
+import messages from './messages';
+
+const onDeleteSubmitMock = jest.fn();
+const closeMock = jest.fn();
+
+const renderComponent = (props) => render(
+  <IntlProvider locale="en">
+    <DeleteModal
+      isOpen
+      close={closeMock}
+      onDeleteSubmit={onDeleteSubmitMock}
+      {...props}
+    />
+  </IntlProvider>,
+);
+
+describe('<DeleteModal />', () => {
+  it('renders DeleteModal component correctly', () => {
```

Comment:
In tests for other components, you used the word "renders" in the singular. Let's stick to this rule.

Reply:
Done